easybuild/easyconfigs/t/Triangle/Triangle-1.6-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'MakeCp'
+
+name = 'Triangle'
+version = '1.6'
+
+homepage = 'http://www.cs.cmu.edu/~quake/triangle.html'
+description = """Triangle generates exact Delaunay triangulations, constrained Delaunay triangulations,
+ conforming Delaunay triangulations, Voronoi diagrams, and high-quality triangular meshes.
+ The latter can be generated with no small or large angles,
+ and are thus suitable for finite element analysis."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.netlib.org/voronoi/']
+sources = ['%(namelower)s.zip']
+checksums = [
+    '1766327add038495fa3499e9b7cc642179229750f7201b94f8e1b7bee76f8480',  # triangle.zip
+    '38fc1395c2392f627068b669275c30da0c25b1a6db8ed8b80d6bf05a98971568',  # Triangle-1.6_makefile.patch
+]
+
+patches = ['%(name)s-%(version)s_makefile.patch']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+]
+
+buildopts = 'triangle trilibrary'
+
+files_to_copy = [
+    (['triangle', 'tricall'], 'bin'),
+    (['triangle.h'], 'include'),
+    (['libtriangle.a'], 'lib'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/triangle', 'bin/tricall', 'include/triangle.h', 'lib/libtriangle.a'],
+    'dirs': []
+}
+
```

Comment:
Is there a sanity check command you could run? (e.g. `triangle --version` or something) If so, please add using the keyword `sanity_check_commands`